src/server.js

```diff
@@ -0,0 +1,16 @@
+const path = require('path');
+const express = require('express');
+
+const app = express();
+const PORT = 3000;
+
+app.use(express.static(path.resolve(__dirname, '../dist')));
+
+app.listen(PORT, () => {
```

Comment:
Может быть такое, что порт 3000 занят другим приложением, было бы классно дать возможность задавать порт через команду запуска nodejs скрипта

```suggestion
app.listen(process.env.PORT || PORT, () => {
```

Reply:
Поправил